### PR TITLE
Enable Qemu command Interface

### PR DESCRIPTION
**Pull request by sharkconi**
1. Switch to qemu command to support ipmi-bmc function
2. release qemu-system-x86_64 v2.6.0
3. test_core_process/ipmi_command_response.py works
4. improve infrasim.conf.example

### CODE REVIEW

**Comment by sharkconi** [comment]
@turtle-fly @fub2 @JuneZhou19 @PaynePei

**Comment by turtle-fly** [comment]
:+1:

File: infrasim/qemu.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        return cmd
+
+def start_qemu():
+    vm = QEMU()
+    vm.read_from_config()
+    run_command(vm.get_qemu_cmd(), True, None, None)
+
+def stop_qemu():
+    qemu_stop_cmd = "pkill qemu"
+    run_command(qemu_stop_cmd, True, None, None)
```
Comment (fub2) [inline infrasim/qemu.py]: :+1:

File: infrasim/qemu.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+class QEMU():
+    VM_DEFAULT_CONFIG = "/etc/infrasim/infrasim.conf"
+    def __init__(self):
+        self.vm_features = {"name": "quanta_d51", "memory": 1024,
+                     "vcpu": 2, "cpu": "", "smbios":"", "kvm":"", "sol":"",
+                     "disks":"", "networks":""}
+        self.vm_templates = {"qemu":"", "disk":"", "net_bridge":"", "net_nat":""}
+        self.start_command = ""
+        self.vm_templates["qemu"] = "/usr/local/bin/qemu-system-x86_64 -name {name} -boot ncd,menu=on -machine pc-q35-2.5 {cpu} {kvm} -m {memory} -realtime mlock=off -smp {vcpu} -rtc base=utc {smbios} -device ahci,id=sata0 {disks} {networks} -vnc :1 {sol} -device ipmi-bmc-extern,chardev=ipmi0,id=bmc0 -device isa-ipmi-kcs,bmc=bmc0 -cdrom /dev/sr0 &"
```
Comment (PaynePei) [inline infrasim/qemu.py]: too long in a line

File: infrasim/qemu.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    def set_network(self, network="nat"):
+        conf = ConfigParser.ConfigParser()
+        conf.read(self.VM_DEFAULT_CONFIG)
+        macs = []
+        if conf.has_option("node", "mac1") is True:
+           macs.append(conf.get("node", "mac1"))
+        if conf.has_option("node", "mac2") is True:
+           macs.append(conf.get("node", "mac2"))
+        if conf.has_option("node", "mac3") is True:
+           macs.append(conf.get("node", "mac3"))
```
Comment (PaynePei) [inline infrasim/qemu.py]: only three mac are suppported?

File: infrasim/qemu.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        self.vm_templates = {"qemu":"", "disk":"", "net_bridge":"", "net_nat":""}
+        self.start_command = ""
+        self.vm_templates["qemu"] = "/usr/local/bin/qemu-system-x86_64 -name {name} -boot ncd,menu=on -machine pc-q35-2.5 {cpu} {kvm} -m {memory} -realtime mlock=off -smp {vcpu} -rtc base=utc {smbios} -device ahci,id=sata0 {disks} {networks} -vnc :1 {sol} -device ipmi-bmc-extern,chardev=ipmi0,id=bmc0 -device isa-ipmi-kcs,bmc=bmc0 -cdrom /dev/sr0 &"
+        self.vm_templates["disk"] = "-drive file={file},format=qcow2,if=none,id=drive-sata0-0-{idx} -device ide-hd,bus=sata0.0,drive=drive-sata0-0-{idx},id=sata0-0-{idx} "
+        self.vm_templates["net_bridge"] = "-net nic,model=e1000,macaddr={mac} -net tap,id=hostnet0,fd={fd} {fd}<>/dev/tap{tap} "
+        self.vm_templates["net_nat"] = "-netdev user,id=vnet{id} -device e1000,mac={mac},netdev=vnet{id} "
+
+        #self.set_smbios()
+
+    def __set_default_config(self):
```
Comment (PaynePei) [inline infrasim/qemu.py]: what's this function used for?

**Pull request merged**